lib/src/presentation/widgets/home_page_widgets/movie_grid_information.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+
+import '../../../core/utils/styles.dart';
+import '../../../domain/entities/genre.dart';
+import '../../../domain/entities/movie.dart';
+import '../../pages/detail_movie_page.dart';
+import '../general_widgets/circle_reveal_clipper.dart';
+
+class MovieGridInformation extends StatefulWidget {
+  MovieGridInformation({
+    super.key,
+    required this.movie,
+    required this.genres,
+  });
+
+  final Movie movie;
+  final List<Genre> genres;
+
+  @override
+  State<MovieGridInformation> createState() => _MovieGridInformationState();
+}
+
+class _MovieGridInformationState extends State<MovieGridInformation> {
+  bool _selected = false;
+  double animationHeight = 290;
+  double animationWidth = 200;
```

Comment:
If you define these as const, you can make the dependant variables non-`late`. I think you're not changing these values internally, right?

Reply:
You are right! Fixed it!